nidmfsl/fsl_exporter/fsl_exporter.py

```diff
@@ -485,8 +487,60 @@ def _find_inferences(self):
                 labels, num_feat = scipy.ndimage.label(excset_img.get_data(),
                                                        structure)
 
-                # Relabel using new sets of labels (times 10000)
-                labels = labels*max(num_feat, 10000)
+                # Update labels to match FSL's table
+                # If clusters are available in voxel space
+                cluster_vox_file = glob.glob(
+                    os.path.join(analysis_dir,
+                                 'cluster*' + str(stat_num) + '.txt'))
+                if not cluster_vox_file:
+                    cluster_vox_tab = None
+                elif len(cluster_vox_file) > 1:
```

Comment:
I am confused here - is there not a `cluster_vox_file` per `cluster_vox_tab`?

Reply:
Line 497 is checking that we found exactly one file for the cluster table (in voxel space). Does this help?

Reply:
I think so... Could you give an example scenario where there would be more than one file for the cluster table?

Reply:
This is error catching: no scenario in mind but if this happens then the code will be able to handle it rather than crashing.